transbank/webpay/webpayplus/options.py

```diff
@@ -65,8 +63,13 @@ def build_options(options):
         base_url = WebpayPlus.integration_type_url()
 
         if options is not None:
-            commerce_code = options.commerce_code
-            api_key = options.api_key
-            base_url = WebpayPlus.integration_type_url(options.integration_type)
+            if hasattr(options, 'commerce_code'):
```

Comment:
Why do you have to do this? Why the option object might not have the fields set?